static/js/components/ErrorMessage_test.js

```diff
@@ -111,10 +108,32 @@ describe("ErrorMessage", () => {
     });
 
     describe('dashboard page', () => {
+      const dashboardSuccessActions = [
+        REQUEST_DASHBOARD,
+        RECEIVE_DASHBOARD_SUCCESS,
+        REQUEST_COURSE_PRICES,
+        RECEIVE_COURSE_PRICES_SUCCESS,
+        REQUEST_GET_USER_PROFILE,
+        RECEIVE_GET_USER_PROFILE_SUCCESS,
+        REQUEST_GET_PROGRAM_ENROLLMENTS,
+        RECEIVE_GET_PROGRAM_ENROLLMENTS_SUCCESS,
+      ];
```

Comment:
this is defined in 3 different places. can you just defined this and export it in `DashboardPage_test.js` or something like that?